darts/timeseries.py

```diff
@@ -5659,6 +5659,39 @@ def concatenate(
     return TimeSeries.from_xarray(da_concat, fill_missing_dates=False)
 
 
+def intersect(series: Sequence[TimeSeries]):
```

Comment:
Adding the return type in the function signature is always preferred

```suggestion
def slice_intersect(series: Sequence[TimeSeries]) -> Sequence[TimeSeries]:
```